template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Before you start guide</td>
+    <td>This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide. If this documentation type has certain key decisions that need to be made with stakeholders or if it has a dependency on some other type of document or process, that should be mentioned in this guide.</td>
+    <td>No, optional</td>
+  </tr>
+  <tr>
+    <td>Template example</td>
+    <td>The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, it is optional to provide an example of the template. However, if you want to create a fake example as part of your template set, you may do so.</td>
+    <td>No, optional</td>
+  </tr>
+</table>
+
+For more detailed information about each deliverable, see [Template Deliverables](template-deliverables.md).
+
+
+## Overview of the template writing phases
+Contributing a template project to our repository has five phases, as summarized in this table:
+
+<table>
+  <tr>
+    <th>Phase</th>
+    <th>Your goals</th>
+    <th>Resources to help you</th>
+  </tr>
+  <tr>
+    <td>1 - Join the community</td>
+    <td><ul>
+          <li>Join our project’s communication channels.</li>
+          <li>Decide which initiative you’d like to work on based on your interests and experience.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>The Good Docs Project welcoming committee</li>
+          <li>Working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>2 - Adopt a template</td>
+    <td><ul>
+          <li>Join a template writing working group led by a template mentor.</li>
+          <li>Work with a template mentor and your working group to decide which template you’ll work on.</li>
+          <li>Assign yourself to the corresponding issue for that template.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>The issues board on the templates repo</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+          <li>Template working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>3 - Research and draft</td>
+    <td><ul>
+          <li>Research examples and best practices for the type of template you are working on.</li>
+          <li>Collaborate and get early feedback on your research from your template mentor and/or other templateers as part of a template writing working group.</li>
+          <li>Create drafts of the template set deliverables in Google Docs or your preferred tool.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>Working groups</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>4 - Workshop the drafts with the community</td>

```

Comment:
I'd change this to "Revise the drafts based on community review"

Reply:
See my reply to your previous comment.

Reply:
I'm adopting your suggestion, per our discussion in last week's general meeting. Resolved: We had a lengthy discussion about it at our last general meeting. The consensus was that the term "workshop," although accurate, was not necessarily understood across all cultures. For that reason, we're going to change the name of this phase to "Get feedback on drafts from the community." There was also some confusion about who could review these drafts. We'll clarify that community reviewers includes anyone who is a member of the Good Docs Project, including members of template writing working groups. For now, we'll keep the requirement to have at least three community reviews are required to move to the next phase. We'll adjust this requirement if we discover it is too difficult.